__tests__/intergration.tests.js

```diff
@@ -180,6 +177,41 @@ describe("/api", () => {
   });
 });
 
+describe("GET /api/users/:username", () => {
+  test("GET: 200 returns a single user object with the correct properties", () => {
+    return request(app)
+      .get("/api/users/butter_bridge")
+      .expect(200)
+      .then((response) => {
+        expect(response.body.user.length).toBe(1);
+        expect(response.body.user[0]).toEqual({
+          username: "butter_bridge",
+          name: "jonny",
+          avatar_url:
+            "https://www.healthytherapies.com/wp-content/uploads/2016/06/Lime3.jpg",
+        });
+      });
+  });
+
+  test("GET: 404 returns an error message when the username is invalid", () => {
+    return request(app)
+      .get("/api/users/`DELETE * FROM users where user.username=lurker`")
+      .expect(404)
+      .then((response) => {
+        expect(response.body.message).toBe("Not Found");
+      });
+  });
```

Comment:
Intriguing url for this test, but watch out because you're muddying the waters between several different tests here. Consider the followig=ng

- a URL passed in your .get with an SQL command -  this looks like an sql injection test
- expecting a 404 - That's an error code for a not found user
- '...the username is invalid' - In the test description suggests a 500 test, which usually means the wrong datatype has been use. For a 400 we'd instead say something along the lines of 'valid but not found'.

focus this test more onto EXACTLY what you're after, and consider making the sql